File: app/src/main/java/com/example/dev/hazikura/fragment/Management/MyListItem.java
package com.example.dev.hazikura.fragment.Management;

import android.util.Log;

/**
 * Created by yokoro
 **/

public class MyListItem {

    protected int id;
    protected String date;
    protected String content;
    protected String number;
    protected String amount;

    public MyListItem(int id, String date, String content, String amount){
        this.id = id;
        this.date = date;
        this.content = content;
        this.amount = amount;
    }

    public MyListItem(int id, String date, String content, String number, String amount){
        this.id = id;
        this.date = date;
        this.content = content;
        this.number = number;
        this.amount = amount;
    }

    public int getId(){
        Log.d("取得ID：", String.valueOf(id));
        return id;
    }

    public String getDate() {
        return date;
    }

    public String getContent() {
        return content;
    }

    public String getNumber(){ return number; }

    public String getAmount() {
        return amount;
    }

}

File: app/src/main/java/com/example/dev/hazikura/fragment/management/MyListItem.kt
package com.example.dev.hazikura.fragment.management

import android.util.Log

/**
 * Created by yokoro
 */

class MyListItem {

    private var itemId: Int = 0
    var date: String
        private set
    var content: String
        private set
    lateinit var number: String
        private set
    var amount: String
        private set

    constructor(id: Int, date: String, content: String, amount: String) {
        this.itemId = id
        this.date = date
        this.content = content
        this.amount = amount
    }

    constructor(id: Int, date: String, content: String, number: String, amount: String) {
        this.itemId = id
        this.date = date
        this.content = content
        this.number = number
        this.amount = amount
    }

    fun getId(): Int {
        Log.d("取得ID：", itemId.toString())
        return itemId
    }

}
